index.js

```diff
@@ -94,6 +94,41 @@ module.exports.init = function (func) {
       }
     };
 
+    var arpeggiate = function (obj, start, duration, callback) {
+      var notes;
+
+      start = ctx.currentTime + (start || 0);
+      duration = duration || defaultDuration;
+
+      if (callback) callback = _.partial(callback, obj);
+
+      // For a chord, play every note in succession
+      if (s11.chord.isChord(obj)) {
+        notes = ensureOctave(obj).chord;
+        _.each(notes, function (note, i) {
```

Comment:
Since this `_.each` block is the same in every case, could you move it out of the conditionals?